packages/cli/package.json

```diff
@@ -26,9 +26,12 @@
   "dependencies": {
     "@oclif/core": "^1.20.4",
     "@remix-run/dev": "1.12.0",
+    "@remix-run/react": "1.12.0",
     "@shopify/cli-kit": "3.29.0",
     "@shopify/mini-oxygen": "^1.3.1",
     "fast-glob": "^3.2.12",
+    "@shopify/remix-oxygen": "^1.0.0-alpha.3",
+    "@shopify/storefront-kit-react": "^2023.1.3",
```

Comment:
These might be better as devDependencies + peerDependencies. The user will have them installed for the main app, right?